### PR TITLE
fix point value object sample code and point type sample code

### PR DESCRIPTION
**Pull request by Mehrdad-Dadkhah**
I use these codes.
They work wrong because it exchange lat and long while saving and also it has problem when reading from database. since we have wrong data it resolve this problem by exchange them again in convertToPHPValue function!
It is not a good behavior. I fix it.

### CODE REVIEW

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst [hunk 1]
```diff
@@ -176,7 +184,7 @@ Now we're going to create the ``point`` type and implement all required methods.
             return true;
         }
 
-        public function convertToPHPValueSQL($sqlExpr, AbstractPlatform $platform)
```
Comment (mikeSimonson) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: @Mehrdad-Dadkhah Why are you removing the type hint ?

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst [hunk 2]
```diff
@@ -92,9 +92,10 @@ The point class:
 .. code-block:: php
 
     <?php
-
```
Comment (mikeSimonson) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: @Mehrdad-Dadkhah Please don't remove and add spaces randomly in your PR

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst [hunk 3]
```diff
@@ -135,11 +144,10 @@ Now we're going to create the ``point`` type and implement all required methods.
     <?php
 
     namespace Geo\Types;
-
+
```
Comment (mikeSimonson) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: spaces ?

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst [hunk 4]
```diff
@@ -92,9 +92,10 @@ The point class:
 .. code-block:: php
 
     <?php
-    
+
     namespace Geo\ValueObject;
 
+    use Assert\Assertion;
```
Comment (mikeSimonson) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: Can you remove that use that is not used anywhere

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst [hunk 5]
```diff
@@ -135,11 +144,10 @@ Now we're going to create the ``point`` type and implement all required methods.
     <?php
 
     namespace Geo\Types;
-
+    
     use Doctrine\DBAL\Types\Type;
     use Doctrine\DBAL\Platforms\AbstractPlatform;
-
-    use Geo\ValueObject\Point;
+    use App\web\one\Domain\Restaurant\Deliveryzone\Model\valueObjects\Point;
```
Comment (mikeSimonson) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: Please keep the namespace from the example as it's used in the other files

**Review by mikeSimonson — CHANGES_REQUESTED** [review]
Thanks for taking the time to improve the documentation. Can you update your PR with my remarks ?

Thanks

File: en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst (hunk 5 above)
Comment (Mehrdad-Dadkhah) [inline en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst]: oops! excuse me
done

**Comment by Mehrdad-Dadkhah** [comment]
@mikeSimonson 
Thanks for taking the time to improve my changes.

I do.

**Review by mikeSimonson — APPROVED** [review]

**Comment by mikeSimonson** [comment]
@Mehrdad-Dadkhah Thanks

**Pull request merged**